index.html

```diff
@@ -6,21 +6,30 @@
   </head>
   <body>
     <h1>Welcome to the Salmon Cookies Website</h1>
-    <img src="https://raw.githubusercontent.com/codefellows/seattle-201d19/c22e563a40184ee90e8a1cfb930f6aa7e65b3571/class-06-dom-objects-domain_modeling/assets/salmon.png">
-      <div id = "leftmenu">
+    <img src="C:\Users\Isak Swearingen\Pictures\Saved Pictures\salmon.png">
```

Comment:
This filepath is relative to your computer, and your computer only. It won't work on mine or anyone elses. You need to save the picture into your repo, then put a relative filepath from the html file to the image file inside the src attribute of the image. Let me know if you have questions!